lib/commands/command.bash

```diff
@@ -8,17 +8,49 @@ if [ -z "$(declare -f -F watch_file)" ]; then
   eval "$(asdf exec direnv stdlib)"
 fi
 
+# This is inspired by https://stackoverflow.com/a/1116890
```

Comment:
just for context, here's the function that asdf itself uses: 
https://github.com/asdf-vm/asdf/blob/master/lib/utils.bash#L450

can't remember if we have the asdf lib loaded while running this file, but if so, we could maybe re-use that?

Reply:
Oh by bad haha I see the purpose of this is actually to load asdf utils :D

Reply:
Ooh, would you prefer that I copy that implementation?

Reply:
Nevermind, now that we're using the `asdf info` trick, we don't need to resolve any symlinks :partying_face: